app/templates/events_nav.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html"%}
+
+
+{% block app_content %}
+    {% block content_pageHeading %}
+    <div class="text-center">
+        <h1 id="title">{{page_title}}</h1>
+        {% if eventData.name != 'All Volunteer Training' and eventData['program'] == None %}
+            <h3>CELTS-Sponsored Event</h3>
+        {% elif eventData['program'] %}
+            <h3>{{eventData["program"].programName}} Event</h3>
```

Comment:
This should not appear when creating an event:

![image](https://github.com/BCStudentSoftwareDevTeam/celts/assets/26262045/8b0c7a4d-3fb8-4abf-8111-d5ed18775507)